### PR TITLE
Table refactor

### PR DESCRIPTION
**Pull request by Tapiiri**
Some refactors to the Table component, details in commits.

### CODE REVIEW

**Comment by Tapiiri** [comment]
Test

File: src/components/Table.js [hunk 1]
```diff
@@ -1,57 +1,57 @@
...
 import "../App.css";
+import { DEFAULT_NUMBER_OF_SEATS } from "../constants/components";
 import { array, number, bool } from "prop-types";
 
 import Seat from "./Seat";
 
-const mapStateToProps = state => ({ // eslint-disable-line
+const mapStateToProps = state => ({
+  // eslint-disable-line
   // eslint-disable-line
```
Comment (VonLatvala) [inline src/components/Table.js]: MIKÄ VITTU TÄÄ ON

**Review by VonLatvala — COMMENTED** [review]
4 spaceee bruv

**Review by Earlo — APPROVED** [review]
Fooking brilliant m8

**Pull request merged**